### PR TITLE
fix typos in 'How to use' section.

### PR DESCRIPTION
**Pull request by chasingSublimity**
Hey!

Type: bug fix | new feature | code quality | documentation

Link to issue:

This pull request affects the following components: **(please check boxes)**

* [ ] Core
* [ ] Analyzer
* [ ] Compiler
* [ ] Control Flow Graph
* [x] Documentation

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](/.github/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [ ] I wrote some tests for this PR.

Small description of change:

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ Notice:  Constant BBBB does not exist in self scope in fixtures/simple/undefined
 
     return self::BBBB; 
 
-Notice:  You are trying to cast 'string' to 'string'. in fixtures/simple/code-smell/StandardFunctionCall.php on 16 [stupid.cast]
+Notice:  You are trying to cast 'string' to 'string' in fixtures/simple/code-smell/StandardFunctionCall.php on 16 [stupid.cast]
```
Comment (K-Phoen) [inline README.md]: This message is generated by PHPSA, we should also fix it [in the analyzer](https://github.com/ovr/phpsa/blob/49827aed3dc3b425b862ea99762953c42cd70fc5/src/Analyzer/Pass/Expression/Casts.php#L50)

File: README.md [hunk 2]
```diff
@@ -46,9 +46,9 @@ Notice:  Constant BBBB does not exist in self scope in fixtures/simple/undefined
...
 
-    return (string) json_encode(array(
+    return (string) json_encode(array)
```
Comment (K-Phoen) [inline README.md]: This is the line triggering the notice so `return (string) json_encode(array(` is actually correct.

Comment (chasingSublimity) [inline README.md]: Derp. Good catch!

**Comment by ddmler** [comment]
Thanks @chasingSublimity i merge this one, please fix the issue in pr 190

**Pull request merged**